src/screens/HomeScreen/components/SearchCarousel.tsx

```diff
@@ -0,0 +1,61 @@
+import React, {useState} from 'react';
+import {Carousel} from '../../../components';
+import {Dimensions, Image, StyleSheet, View} from 'react-native';
+import {verticalScale} from '../../../utils';
+import Dots from 'react-native-dots-pagination';
+import {CarouselDataItem} from './CarouselDataItem/CarouselDataItem';
+import {Pallete} from '../../../theme';
+import {SearchCarouselData} from '../../../assets/data';
+
+export const SearchCarouselRenderItem = ({index, item}: any) => (
+  <View style={styles.renderItem} key={index}>
```

Comment:
instead of styles.renderItem, it should be styles.itemContainer

Reply:
Fixed.